Vagrantfile

```diff
@@ -7,6 +7,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = "debian/stretch64"
   config.vm.network :forwarded_port, guest: 80, host: 8080
   config.vm.synced_folder '.', '/vagrant', disabled: true
+
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 2048
```

Comment:
Should we not reduce the number of language indexed in dev instead? Do we need all of them all the time?
It might be useful to figure out corner cases here and there but that could be part of a test suite maybe?

Reply:
I agree. Increasing RAM here was more of a temporary solution to get things working. I opened a ticket for that: Tatoeba/tatoeba2#1689